app/src/androidTest/java/tech/pauly/findapet/shared/MainTabActivityTest.java

```diff
@@ -47,14 +47,4 @@ public void onClickFavorites_seeOnlyFavorites() {
         RobotUtils.doesNotSeeView(R.id.fragment_shelters);
         RobotUtils.doesNotSeeView(R.id.fragment_settings);
     }
-
```

Comment:
- Removed Settings tab, will likely move it to the toolbar in the favorites tab: https://www.pivotaltracker.com/story/show/157534780